devel/Arduino/Portfile

```diff
@@ -17,6 +15,9 @@ platforms           darwin
 maintainers         {gmail.com:giansalvo.gusinu @giansalvo} openmaintainer
 license             GPL-2
 
+# I don't know what to set here
+#supported_archs         noarch
+
```

Comment:
In general, `supported_archs         noarch` is for scripts like a python or perl script because they are essentially platform independent. If there is anything compiled, then this should not be used.

Reply:
Thanks.
In Arduino's build system all these "platforms" are said to be all rigth: i386, amd64, arm, aarch64, linux32, linux64, linuxarm.
How do I translate these settings to the portfile?

Reply:
you need to use the terms MacPorts understands. So select from i386, x86_64 and ppc as appropriate. Note, i386 is 10.6 only, and ppc 10.5 and older...

Reply:
i386 is also for universal builds. Personally I would leave this out altogether. We only need it if some arch is known to be broken or if it's noarch.